src/lib.rs

```diff
@@ -229,8 +268,8 @@ impl NetworkResource {
                                 channel.send(Ok(Packet::copy_from_slice(packet.payload())))
                             }) {
                             Some(Ok(())) => false,
-                            Some(Err(error)) => {
-                                log::error!("Server Send Error: {}", error);
+                            Some(Err(CrossbeamSendError(_packet))) => {
```

Comment:
If it is used it should be `packet` not `_packet`.

Reply:
it's not used in the log message any more. it's a rare case as long as dropped connections are cleaned up, and the packet isn't lost - it's delivered on a newly created channel. so it seemed spammy to log the packet contents.

Reply:
Oh. Right. I misunderstood the change.